static.go

```diff
@@ -149,7 +149,15 @@ type Frequency struct {
 	ExactTimes ExactTimes
 }
 
-type ParseStaticOptions struct{}
+type StopOptions struct {
```

Comment:
should we just put the InheritWheelchair boarding field directly on the ParseStaticOptions struct, and remove the StopOptions struct? I feel it's unambiguous there, I may be wrong though

Reply:
I think that makes sense - I've remove the `StopOptions` struct.